bin/cron/zendesk_slack_report

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env ruby
+
+#
+# Zendesk -> Slack ticket count integration
+#
+# This script is used to notify various rooms in Slack about how many Zendesk
+# tickets are open in their associated groups.
+#
+# It uses the Search feature of the Zendesk Support API, documented here:
+# https://developer.zendesk.com/rest_api/docs/support/search
+#
+# This the following settings to be configured in locals.yml or globals.yml:
+# - hip_chat_logging: true
+# - slack_endpoint: <the unique part of your webhook URL>
+# - slack_token: <your bot user token>
+# - zendesk_dev_token: <your API token>
+#
+
+require 'httparty'
+require 'uri'
+require_relative '../../deployment'
+require 'cdo/chat_client'
+
+# CONFIGURATION
+
+# Statuses to report on
+STATUSES_TO_REPORT = %w(new open)
+
+# Slack room -> Zendesk group mappings to report on
+MONITORING_GROUPS = {
+  # "slack-room-name" => ["Zendesk group name", "Zendesk group name", ...]
+  "learning-platform" => ["Learning Platform"],
+  # "developers" => ["Developers"],
+  # "plc-tools" => ["PLC Tools"],
+  # "privacy" => ["Privacy"],
+  # "star-labs-cabal" => ["*Labs"],
+  # "user-accounts" => ["Accounts"],
+}
+
+# Zendesk group -> Zendesk filter id mappings for helpful links (optional)
+# rubocop:disable Style/NumericLiterals
+GROUP_FILTERS = {
+  "Developers" => 47222126,
+  "Learning Platform" => 360082967212,
+  "PLC Tools" => 114103940251,
+  "*Labs" => 301129028,
```

Comment:
The filter ids here override the Zendesk link posted for a particular group.  If one is not found here (e.g. "Accounts") we will link to the generic Zendesk search page instead.